src/main/java/htsjdk/variant/variantcontext/writer/VariantContextWriterBuilder.java

```diff
@@ -468,7 +468,7 @@ else if (STREAM_TYPES.contains(this.outType))
      * attempt to resolve any symlinks and try again.  If that fails, and the output file exists
      * but is neither a file or directory then VCF_STREAM is returned.
      */
-    protected static OutputType determineOutputTypeFromFile(final File f) {
+    public static OutputType determineOutputTypeFromFile(final File f) {
```

Comment:
Can you add a test + javadoc for this newly-public method?

Reply:
Looks like there is a test in VariantContextWriterBuilderUnitTest.